src/acreplace.rs

```diff
@@ -1,22 +1,64 @@
-use aho_corasick::AhoCorasickBuilder;
-use aho_corasick::AhoCorasick;
+use aho_corasick::{AhoCorasick, AhoCorasickBuilder, MatchKind};
 use std::{
     cell::RefCell,
     collections::hash_map::HashMap
 };
+use serde::Deserialize;
 
 struct Replacements {
     pub automaton: AhoCorasick,
-    pub replacements: Vec<String>
+    pub replacements: Vec<String>,
+}
+
+#[derive(Deserialize)]
+struct AhoCorasickOptions {
+    #[serde(default, deserialize_with = "deserialize_byond_bool")]
+    pub anchored: bool,
+    #[serde(default, deserialize_with = "deserialize_byond_bool")]
+    pub ascii_case_insensitive: bool,
+    #[serde(default, deserialize_with = "deserialize_matchkind")]
```

Comment:
Not using serde's remote derive here as I feel it looks messy and the underlying crate receiving new `MatchKind`s would cause compile errors in rust-g, whereas with this new `MatchKind`s are merely not supported without someone adding them.

Reply:
Hesitant :+1:--adding new `MatchKind` without `non_exhaustive` is a breaking change anyway, so we shouldn't be just updating entire major versions frivolously.